llama-index-integrations/storage/docstore/llama-index-storage-docstore-azure/README.md

```diff
@@ -0,0 +1 @@
+# LlamaIndex Docstore Integration: Azure Table Storage
```

Comment:
Do we want to give some install/usage example in the readme? This will show up on `https://llamahub.ai`